src/rotating_gate.c

```diff
@@ -740,14 +741,15 @@ static u8 RotatingGate_CreateGate(u8 gateId, s16 deltaX, s16 deltaY)
 
     template.tileTag = gate->shape + ROTATING_GATE_TILE_TAG;
 
-    spriteId = CreateSprite(&template, 0, 0, 0x94);
+    spriteId = CreateSprite(&template, 0, 0, 0x93);
```

Comment:
Why was this lowered?

Reply:
If I remember right, it was so that shadows would be below the gate